website/source/layouts/docs.erb

```diff
@@ -318,6 +312,24 @@
         </ul>
       </li>
 
+      <li<%= sidebar_current("docs-acl") %>>
+        <a href="/docs/acl/index.html">Access Control (ACLs)</a>
```

Comment:
Would we want other security related things under here in the future? 

TLS bootstrapping information
Serf Encryption
Others?

The nomad guides currently have all of that information under a "Security and Governance" section. Should we do something similarly or do you think its better to keep ACLs separate from that other stuff? I don't have a strong opinion either way, just pointing out a difference between our docs and nomad.

Reply:
I am definitely thinking we need a "Security" category in the future for a couple reasons, 1) Nomad has a security category and it's good to be consistent across products and 2) security is a huge concern for our users. 

I opted to keep ACLs separate because it (being a security feature) is super important for users, which was stated in the previous point. However, unlike security in general, it already has a large (relatively speaking) number of docs that are associated with it and will continue to grow in the near future with RB's work. Lastly, several months ago the ACL guide was identified as being the most popular guide using Google Analytics. Surfacing it to the side nav will make it easier to find.